src/HonzaBotner.Discord.Services/Commands/VoiceCommands.cs

```diff
@@ -9,141 +9,140 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 
-namespace HonzaBotner.Discord.Services.Commands
+namespace HonzaBotner.Discord.Services.Commands;
+
+[Group("voice")]
+[Description("Commands to control (and only works in) custom voice channels.")]
+[RequireGuild]
+public class VoiceCommands : BaseCommandModule
 {
-    [Group("voice")]
-    [Description("Commands to control (and only works in) custom voice channels.")]
-    [RequireGuild]
-    public class VoiceCommands : BaseCommandModule
+    private readonly IVoiceManager _voiceManager;
+    private readonly CustomVoiceOptions _voiceConfig;
+    private readonly ILogger<VoiceCommands> _logger;
+
+    public VoiceCommands(IVoiceManager voiceManager, IOptions<CustomVoiceOptions> options,
+        ILogger<VoiceCommands> logger)
     {
-        private readonly IVoiceManager _voiceManager;
-        private readonly CustomVoiceOptions _voiceConfig;
-        private readonly ILogger<VoiceCommands> _logger;
+        _voiceManager = voiceManager;
+        _voiceConfig = options.Value;
+        _logger = logger;
+    }
 
-        public VoiceCommands(IVoiceManager voiceManager, IOptions<CustomVoiceOptions> options,
-            ILogger<VoiceCommands> logger)
-        {
-            _voiceManager = voiceManager;
-            _voiceConfig = options.Value;
-            _logger = logger;
-        }
+    [Command("add")]
+    [Aliases("new")]
+    [Description("Create new voice channel. Users has 30 seconds to join.")]
+    [Priority(2)]
+    public async Task AddVoiceChannelWithLimitAndPublic(
+        CommandContext ctx,
+        [Description("Name of the channel.")] string name,
+        [Description("Limit number of members who can join. Use 0 to set to unlimited.")]
+        int limit = 0,
+        [Description("Set voice channel public.")]
```

Comment:
The formating changes here and on line 68 of the same file contradict each other

Reply:
We have the .editorconfig file that should set up everything. Not sure why the format tool chooses different styles, especially with annotations.